feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -76,20 +83,24 @@ export class CustomerEmulationComponent implements OnInit, OnDestroy {
   }
 
   openCustomer360() {
-    const data = { customer: this.customer };
-    this.launchDialogService?.openDialogAndSubscribe(
-      LAUNCH_CALLER.ASM_CUSTOMER_360,
-      this.customer360LauncherElement,
-      data
-    );
+    this.isCustomer360Loaded$
+      .pipe(filter((isReady) => Boolean(isReady)))
+      .subscribe(() => {
+        const data = { customer: this.customer };
```

Comment:
When the main ASM emulates a customer to retrieve customer information, it will subsequently switch the UI to display the 'Customer 360' button, so there would not be a race condition.

Reply:
I was wrong.  I added a condition to the template